doc/languages-frameworks/python.md

```diff
@@ -422,7 +422,7 @@ exists.
 
 #### Missing `tkinter` module standard library
 
-To reduce closure size the `Tkinter`/`tkinter` module is available as `python.tkinter`.
+To reduce closure size the `Tkinter`/`tkinter` is put in a separate output. The `tkinter` is available as `python35Packages.tkinter`.
```

Comment:
In these docs I mix a lot between the different sets. The idea behind it is that the reader sees that these things are valid for all interpreter versions. However, maybe it is better to just always stick with `pythonPackages` and mention only once that other sets can be used as well.